### PR TITLE
2 add readme

### PR DESCRIPTION
**Pull request by arielleleon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,76 @@
 # Multiplane optical physiology processing pipeline
+
+
+# aind-multiplane-ophys-pipeline
```
Comment (dyf) [inline README.md:4]: can probably remove

File: README.md [hunk 2]
```diff
@@ -1 +1,76 @@
...
+
+The multiplane-ophys pipeline is a processing pipeline which uses [Suite2p](https://github.com/MouseLand/suite2p) for both motion correction and segmentation. The final outputs of the pipeline are ROI events detected by [OASIS](https://github.com/j-friedrich/OASIS).
```
Comment (dyf) [inline README.md:6]: I would emphasize that this is for data in which multiple planar fields of view are collected simultaneously and maybe link to the mesoscope docs.

Comment (dyf) [reply README.md:6]: You could also link to the single-plane pipeline here.

File: README.md [hunk 3]
```diff
@@ -1 +1,76 @@
...
+All data within the following HDF5 files are stored under the 'data' key as a numpy array.
+
+* Decrosstalked movie (HDF5)
+* Decrosstalked episodic mean field of view (HDF5)
+* Decrosstalked eposodic mean field of view registered to pair (HDF5)
+* Episodic mean field of view of the registered movie (HDF5)
+
+**`extraction`**
+
+* Extraction (HDF5): Datasets include:
```
Comment (dyf) [inline README.md:57]: `extraction.h5`: HDF5 file including:

File: README.md [hunk 4]
```diff
@@ -1 +1,76 @@
...
+
+* `aind`: data intgestion used at AIND. The input folder must contain a subdirectory called `pophys` (for planar-ophys) which contains the raw TIFF timeseries. The root directory must contain JSON files following [aind-data-schema](https://github.com/AllenNeuralDynamics/aind-data-schema).
+
+# Output
+
+The output pipeline is saved to the `results` with JSON files following [aind-data-schema](https://github.com/AllenNeuralDynamics/aind-data-schema). Each parallel process will output a folder within the processes field of view (or plane). As the movies go through the processsing pipeline, a JSON file called processing.json gets created and processing data from processing parameters are appended to this file. The final JSON will sit at the root of the `results` folder at the end of processing. 
+
+Tools used to read files in python are h5py, json and csv.
+
+The following folders will be under the field of view directory within the `results` folder:
```
Comment (dyf) [inline README.md:35]: you could make this look like this: 
image

**Review by dyf — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1 +1,227 @@
...
```
Comment (dyf) [inline README.md:36]: small request: add another emoji-based folder display here that shows what the top level folders are

Comment (dyf) [reply README.md:36]: kind of like an index of the below folder descriptions.

**Review by dyf — CHANGES_REQUESTED** [review]

**Review by j-friedrich — COMMENTED** [review]
Figure 1:
Error in Tiff structure for upper panel: the second-last Page is 2N-1, the last one 2N.

Motion Correction:
anatomical_region_registered.h5 is listed twice as output of motion_correction.

Extraction:
The extraction.h5 outputs now much more info, almost too much, see its [README](https://github.com/AllenNeuralDynamics/aind-ophys-extraction-suite2p)

**Pull request merged**